public/index.html

```diff
@@ -11,6 +11,8 @@
     <noscript>
       <strong>We're sorry but <%= htmlWebpackPlugin.options.title %> doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
     </noscript>
+    <script src="https://unpkg.com/vue@next"></script>
```

Comment:
Is this required? If yes, please provide the details.

Reply:
No sir, this is not required.